app/controllers/reservations_controller.rb

```diff
@@ -282,9 +283,14 @@ def load_basic_resources
     @order = Order.find(params[:order_id])
     # It's important that the order_detail be the same object as the one in @order.order_details.first
     @order_detail = @order.order_details.find { |od| od.id.to_i == params[:order_detail_id].to_i }
```

Comment:
I don't. The change that brought it in is here: 4ad3a411820fd29a364745940cce22023af92623
Tests still look good for me with `@order.order_details.find(params[:order_detail_id])` so I'm going ahead with it.